handlers.js

```diff
@@ -415,28 +916,132 @@
+                                if (lastModifiedHours > 12) {
+                                    var lastModifiedHours = lastModifiedHours - 12
+                                }
+                            } else {
+                                var lastModifiedAmPm = 'AM'
+                            }
+                            var lastModifiedStr = lastModifiedMonth+'/'+
+                                                  lastModifiedDay+'/'+
+                                                  lastModifiedYear+', '+
+                                                  lastModifiedHours+':'+
+                                                  lastModifiedMinutes+':'+
+                                                  lastModifiedSeconds +' '+
+                                                  lastModifiedAmPm
+
+                            var name = encodeURIComponent(results[w].name)
+                            var isdirectory = results[w].isDirectory
+                            //var modified = '4/27/21, 10:38:40 AM'
+                            var modified = lastModified
+                            var filesize = file.size
+                            var filesizestr = humanFileSize(file.size)
+                            var modifiedstr = lastModifiedStr
+                            // raw, urlencoded, isdirectory, size, size as string, date modified, date modified as string
+                            if (rawname != 'wsc.htaccess') {
+                            html.push('<script>addRow("'+rawname+'","'+name+'",'+isdirectory+',"'+filesize+'","'+filesizestr+'","'+modified+'","'+modifiedstr+'");</script>')
+                            }
+                            if (w != results.length - 1) {
+                                w++
+                                sendFileList.bind(this, results)()
+                            } else {
+                                DirRenderFinish.bind(this, results)()
+                            }
+                        }.bind(this), function(error) {
+                            console.log('error reading metadata '+error.code)
+                            if (w != results.length - 1) {
+                                w++
+                                sendFileList.bind(this, results)()
+                            } else {
+                                DirRenderFinish.bind(this, results)()
+                            }
+                        }.bind(this))}
+                    this.setHeader('transfer-encoding','chunked')
+                    this.writeHeaders(200)
+                    this.writeChunk(WSC.template_data )
+                    if (this.request.path != '') {
+                        var html = ['<script>start("'+this.request.path+'")</script>',
+                                    '<script>onHasParentDirectory();</script>']
+                    } else {
+                        var html = ['<script>start("/")</script>']
```

Comment:
i don't see how these html are being used. (you'd need to writeChunk them?)

Reply:
Sorry about that. I just did what I knew and it sometimes gets a little messy. Sorry about that